app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -196,15 +196,17 @@ class AvatarView @JvmOverloads constructor(
     ) {
         when (avatarDisplayType) {
             AvatarDisplayType.UNKNOWN_CORRESPONDENT -> loadUnknownUserAvatar()
+            AvatarDisplayType.USER_AVATAR -> AccountUtils.currentUser?.let { loadAvatar(user = it) }
```

Comment:
Why don't you need to update the state anymore like you were before modifying this line?

Reply:
Why do you need to update the state but other calls to loadAvatar(user: User) don't need to?